lizmap/modules/admin/controllers/config.classic.php

```diff
@@ -205,6 +210,17 @@ public function saveServices()
             $ok = false;
         }
 
+        // Check the WMS Server URL
+        $wmsServerURL = $form->getData('wmsServerURL');
+        jClasses::inc('lizmap~qgisServer');
```

Comment:
we should declare qgisServer as an autoloaded class into module.xml, and avoid to use jClasses.

Reply:
`qgisServer` is also used with `jClasses::getService` https://github.com/3liz/lizmap-web-client/blob/master/lizmap/modules/lizmap/classes/lizmapProject.class.php#L684. can `qgisServer` be an autoloaded classes ?